src/components/OSCALBackMatter.js

```diff
@@ -113,18 +118,15 @@ export default function OSCALBackMatter(props) {
               {resource.rlinks &&
                 resource.rlinks.map((rlink) => (
                   <Chip
-                    icon={
-                      getAbsoluteUrl(rlink.href, props.parentUrl).startsWith(
-                        "http"
-                      ) ? (
-                        <OpenInNewIcon />
-                      ) : null
-                    }
+                    icon={<OpenInNewIcon />}
                     key={rlink.href}
                     label={getMediaType(rlink)}
                     component="a"
                     role="button"
-                    href={getAbsoluteUrl(rlink.href, props.parentUrl)}
+                    href={getAbsoluteUrl(
+                      rlink.href,
+                      props.parentUrl.startsWith(".") ? null : props.parentUrl
```

Comment:
Do we still need this?

Reply:
Are you suggesting to remove the `getAbsoluteUrl` call here? I do not think that `rlink.href` is an absolute url so I feel like we may still need to keep the function call.